src/timestamp.ts

```diff
@@ -141,12 +142,10 @@ export class Timestamp extends LongWithoutOverridesClass {
     return new Timestamp({ t, i });
   }
 
-  /** @internal */
-  [Symbol.for('nodejs.util.inspect.custom')](): string {
-    return this.inspect();
-  }
-
-  inspect(): string {
-    return `new Timestamp({ t: ${this.getHighBits()}, i: ${this.getLowBits()} })`;
+  inspect(depth?: number, options?: unknown, inspect?: InspectParameterFn): string {
+    inspect ??= defaultInspect;
+    const t = inspect(this.getHighBits(), options);
+    const i = inspect(this.getLowBits(), options);
```

Comment:
Let's change this to align with the way `toExtendedJSON` gets the high and low bits. Specifically, it is more aligned with what timestamp is to view each component as an unsigned value, which is what the shift 0 does.

```js
> new BSON.Timestamp(BSON.Timestamp.MAX_VALUE)
new Timestamp({ t: -1, i: -1 })
// should be
new Timestamp({ t: 4294967295, i: 4294967295 })
```